Dockerfile

```diff
@@ -19,8 +19,8 @@ COPY README.md /app/build
 
 RUN RUSTFLAGS="-L /usr/lib64/atlas -C link-args=-lsatlas -ltatlas -llapack" cargo install finalfusion-utils --features=opq
 
-COPY pyproject.toml /app/build
 COPY src /app/build/src
-COPY bonn /app/build/bonn
+COPY pyproject.toml /app/build
+COPY python/bonn /app/build/bonn
```

Comment:
So presumably the build was broken until this change as `src` and `bonn` were in different levels?

Reply:
I'm not sure how the release was fine and where this happened, as I was a bit pressed for time, but when I tried to rebuild it locally `bonn` wasn't available only `_bonn` which is why I did this change.
Even after unziping the wheel only package available was `_bonn`. 
Before the directory structure is:

├── _bonn/ # SRC code files
│ ├── _bonn.so file

Now it's :
├── bonn/ # Source code files
│ ├── _bonn.so file
│ ├── extract.py
│ ├── category_manager.py
│ ├── all bonn.py files

Reply:
After release I'm going to try on our onyx environment if there's any issues before writing to onyx. 
Release shouldn't be an issue in their production environment if there is an issue as the bonn version they're using is 0.1.5 and I've updated for release to 0.1.6.
[version used in ons](https://github.com/ONSdigital/dp-nlp-category-api/blob/develop/pyproject.toml#L26)

Reply:
![image](https://github.com/user-attachments/assets/13041c30-305f-4f0e-a065-8403df090065)
The previous error.